src/components/Pictures.js

```diff
@@ -0,0 +1,23 @@
+import Ausblick from "../images/Ausblick.jpg";
+import Freunde from "../images/Freunde.jpg";
+import See from "../images/See.jpg";
+import StrandAbend from "../images/StrandAbend.jpg";
+import Stranddecke from "../images/Stranddecke.jpg";
+import StrandLesen from "../images/StrandLesen.jpg";
+import Strandliegen from "../images/Strandliegen.jpg";
```

Comment:
what about `import images from "../images"` then say like `images.StrandAbend` where you need it? Does that work better?